docs/9-hvordan-tilby.adoc

```diff
@@ -113,14 +113,14 @@ God bruk av unike identifikatorer for selve dataelementene øker gjenbruksverdie
 
 URLer kan også fungere fint som identifikatorer gitt at de er unike, permanente og hensiktsmessige. URLer som samtidig er identifikatorer for en ressurs (for eksempel et datasett, begrep eller en opplysning) kaller vi URIer (Uniform Resource Identifier).
 
-Se https://www.difi.no/fagomrader-og-tjenester/digitalisering-og-samordning/standarder/referansekatalogen/pekere-til-offentlige-ressurser-pa-nett[Referansekatalogen for IT-standarde]r for anbefalinger om hvordan identifikatorer i form av URIer (pekere til offentlige ressurser) bør utformes.
+Se https://www.digdir.no/standarder/referansekatalogen-it-standardar/1480[Referansekatalogen for IT-standarder] for anbefalinger om hvordan identifikatorer i form av URIer (pekere til offentlige ressurser) bør utformes.
 
 === 12. Publiser oversikt over virksomhetens data
 
 [quote,www.regjeringen.no/id2536870/#punkt_tolv]
 Hvilke data som tilgjengeliggjøres bør være drevet av brukernes behov. For at brukerne skal kunne finne og bruke dine data, må de vite hvilke data du har. Virksomheten bør derfor vedlikeholde og publisere en oversikt over hvilke data de forvalter. Dette gjelder også beskrivelser av datasett som av forskjellige grunner ikke er tilgjengeliggjort.
 
-I https://fellesdatakatalog.brreg.no/[Felles datakatalog] kan du registrere datasett selv om ikke selve datasettet er tilgjengelig ennå. Synliggjøring av beskrivelser i en datakatalog er uansett viktig for at brukerne skal vite at data finnes og hvem som forvalter de. Du kan også publisere oversikten lokalt på virksomhetens egne hjemmesider, men beskrivelsene må samtidig være tilgjengelig i maskinlesbart format i tråd med https://doc.difi.no/dcat-ap-no/[Standard for beskrivelse av datasett og datakataloger] slik at de kan høstes av de nasjonale katalogene.
+På https://data.norge.no/[Data.norge.no] kan du registrere datasett selv om ikke selve datasettet er tilgjengelig ennå. Synliggjøring av beskrivelser i en datakatalog er uansett viktig for at brukerne skal vite at data finnes og hvem som forvalter de. Du kan også publisere oversikten lokalt på virksomhetens egne hjemmesider, men beskrivelsene må samtidig være tilgjengelig i maskinlesbart format i tråd med https://data.norge.no/specification/dcat-ap-no[Standard for beskrivelse av datasett og datakataloger] slik at de kan høstes av de nasjonale katalogene.
```

Comment:
Tittelen har blitt endret i mellomtiden: Standard for beskrivelse av datasett, datatjenester og datakataloger (DCAT-AP-NO)